src/components/HomePage/Forms/SelectInputGenre.tsx

```diff
@@ -1,39 +1,50 @@
-import React, { FC } from 'react';
+import React, { FC, useRef } from 'react';
 import {
-  FormGroup, Input, Label,
+    Input, Label,
 } from 'reactstrap';
 import './Form.scss';
+import { Field, useField } from "formik";
 
-export const SelectInputGenre: FC<{ defaultSelected?: string[],
-    onChangeCallBack: Function }> = ({ defaultSelected, onChangeCallBack }) => {
-  const movieGenres = ['Action', 'Adventure', 'Animation',
-    'Comedy', 'Drama', 'Family',
-    'Fantasy', 'Horror', 'Music',
-    'Mystery', 'Romance', 'Science Fiction',
-    'Thriller'];
+export const SelectInputGenre: FC<{
+    defaultSelected: string[],
+    id: string
+}> = ({ defaultSelected, id }) => {
+    const inputRef = useRef<HTMLInputElement | null>(null);
+    const movieGenres = ['Action', 'Adventure', 'Animation',
```

Comment:
Do we have an API endpoint to retrieve Generes?
If not, let's extract this to file constants.ts

Reply:
No, we don't. Moved to constants.ts in commit 144e7682797a8b327df9c556453f21aad1d10993